resources/metadata/shellExecute.yaml

```diff
@@ -42,8 +42,15 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
-        description: scriptArguments that are needed to be passed to scripts. the scriptArguments list is a flat list and has a positional relationship to the `sources` param.
-          For e.g. The scriptArguments string at position 1 will be considered as the argument(s) for script at position 1 in `sources` list. For multiple arguments for a script please add them as a comma seperated string.
+        description: "scriptArguments that are need to be passed to scripts."
+        longDescription: |-
+          The scriptArguments list is a flat list and has a positional relationship to the `sources` parameter.
+
+          For example, the `scriptArguments` string at position 0 will be considered as the argument(s) for script at position 0 in `sources` list.
+          `--sources ".pipeline/firstScript.sh,.pipeline/secondScript.sh" --scriptArguments "$(first_script_arg),$(second_script_arg)"`
+
+          For multiple arguments for a particular script, please add them as a comma-separated string enclosed in additional quotes, e.g.:
+          `--sources ".pipeline/yourScript.sh" --scriptArguments "\"$(first_arg),$(second_arg)\""`
```

Comment:
could we also add a description for multiple arguements for multiple scripts as well, that will complete all possibilities

Reply:
Make sense. I will update the PR accordingly.